packages/block-library/src/site-title/edit/index.js

```diff
@@ -51,12 +54,21 @@ export default function SiteTitleEdit( {
 		} );
 	}
 
+	const {
+		isBold,
+		isItalic,
+		toggleBold,
+		toggleItalic,
+		onKeyDownToggleFontAppearance,
+	} = __experimentalUseFontAppearanceStyles( { attributes, setAttributes } );
```

Comment:
```suggestion
	} = useFontAppearanceStyles( { attributes, setAttributes } );
```
If adopting the pattern suggested in my comment on the `__experimentalUseFontAppearanceStyles` import, this would need a tweak as well.

Reply:
Done!